sbi/inference/snpe/snpe_base.py

```diff
@@ -189,15 +203,33 @@ def __call__(
                 learning_rate=learning_rate,
                 validation_fraction=validation_fraction,
                 stop_after_epochs=stop_after_epochs,
+                max_num_epochs=max_num_epochs,
                 clip_grad_norm=clip_grad_norm,
             )
 
             # Store models at end of each round.
             self._model_bank.append(deepcopy(self._neural_posterior))
             self._model_bank[-1].neural_net.eval()
 
+            # making the call to get_leakage_correction() and the update of
+            # self._leakage_density_correction_factor explicit here. This is just
+            # to make sure this update never gets lost when we e.g. do not log our
+            # things to tensorboard anymore. Calling get_leakage_correction() is needed
+            # to update the leakage after each round.
+            _ = (
+                self._neural_posterior.get_leakage_correction(
+                    x=self._x_o,
+                    force_update=True,
+                    show_progressbar=self._show_progressbar,
+                ),
+            )
+            self._summary["rejection_sampling_acceptance_rates"].append(
+                self._neural_posterior._leakage_density_correction_factor
```

Comment:
why not pass the above `_` here?

Reply:
no good reason for this, you are right, I'll fix this